frontend/packages/integration-tests-cypress/views/list-page.ts

```diff
@@ -26,6 +26,20 @@ export const listPage = {
         cy.byLegacyTestID('dropdown-button').click();
       });
     },
+    by: (rowFilter: string) => {
+      cy.get('.pf-c-toolbar__content-section').within(() => {
+        cy.byLegacyTestID('filter-dropdown-toggle').click();
+        /* PF Filter dropdown menu items are:
+           <li id="cluster">
+             <a data-test-row-filter="cluster">
+           Tried cy.get(`[data-test-row-filter="${rowFilter}"]`).click() which found the <a /> but said not clickable due to
+           it's css having 'pointer-events: none'.  Tried ...click({force: true}) which did the click but page not reloaded with
```

Comment:
Is this a PF accessibility bug?

Reply:
I don't believe so, the `'pointer-events: none'` is being set in [/frontend/public/components/_filter-toolbar.scss](https://github.com/openshift/console/blob/c5299d1e80f1c802b10dda8135c2169928046e64/frontend/public/components/_filter-toolbar.scss#L3)

Reply:
> I don't believe so, the `'pointer-events: none'` is being set in [/frontend/public/components/_filter-toolbar.scss](https://github.com/openshift/console/blob/c5299d1e80f1c802b10dda8135c2169928046e64/frontend/public/components/_filter-toolbar.scss#L3)

In that case, is this a console accessibility bug? @bipuladh Do you remember why you added `pointer-events: none` in #4765?

Reply:
Not sure, but I think it was causing issues while working with `Dropdown` https://github.com/openshift/console/pull/4765/files#diff-db47992a42e1e70bc69c0b457340626915c70d4b2fe7ad976ad7aa080131e424R224